### ISSUE TITLE
Request for week number widget

### ISSUE THREAD

**Issue by mrath** [hussin/timeboxed-watchface#7]
Hey, I like this watchface, but haven't found one I like yet that has week numbers. Could this be added as a widget? 
If you want, I can fork it and try to implement it - wanna try out Pebble development. :)

**Comment by mrath** [comment]
I had a look at the code, and for me it makes sense to add it as a date format, i.e. in locales.c:
```
static char week_text[] = "W00";
...
case FORMAT_WEEKNUMBER:
  strftime(week_text, sizeof(week_text), "W%V", tick_time);
  snprintf(buffer, buf_size, "%s%s%02d%s%s", week_text, " ", tick_time->tm_mday, SEPARATORS[separator], month);
  break;
```
Which (should) render to e.g. 'W41 14.OCT'

**Comment by hussin** [comment]
Thanks! Sorry for the delayed response. I've implemented this for 4.0, so it should be out soon :)

**Comment by hussin** [comment]
Week numbers are out on v4.0! :D
